### ISSUE TITLE
yo i run the program as administrator, turned off anti-virus. installed the latest versions of .NET core and i see these

### ISSUE THREAD

**Issue by Pepsyah** [Reloaded-Project/Reloaded-II#266]
![grgrgrgrgr](https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/2d2fc1df-d5ea-41af-8797-ab2a11f4691e)
![oiewgoiwhgoi](https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/d37527cd-3461-48af-b175-90b6a49138b9)

pls help me

**Comment by Sewer56** [comment]
Delete the folder `sonicheroes.utils.toner` in your `Mods` folder and let the launcher re-download it (it'll happen post launcher restart).

You'll probably be good then

**Comment by Pepsyah** [comment]
ok it loaded but sonic heroes doesnt run and release still doesnt launch application


https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/18fdc3f9-ced3-4004-a19f-6f3b1d263501

**Comment by Pepsyah** [comment]
> Delete the folder `sonicheroes.utils.toner` in your `Mods` folder and let the launcher re-download it (it'll happen post launcher restart).
> 
> You'll probably be good then

yeah but sonic heroes doesnt launch and reloaded II still doesnt launch application

https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/0ee5e986-6cc9-4160-993c-33a3b2ead687

**Comment by Sewer56** [comment]
Sorry for the wait, had a nap.

Normally you shouldn't need to run R2 as administrator, unless any of the following are true
- The game is installed in a place the regular user account can't access (e.g. Program Files). 
- The game is set to run as administrator (in `Properties` window).

For the launcher problem, I'd just doublecheck the path to the EXE, under the `Edit Application`; and make sure it points to the right EXE. I'd then uncheck `Run as Admin` under the Properties page of `TSonic_win.exe` (yours shows the UAC icon so I assume you enabled it).

In any case, do you have anything unusual in your `scripts` folder inside the game folder? If there's nothing unusual there, I'd just try disabling mods one by one, see where you end up.

**Comment by Pepsyah** [comment]
> Delete the folder `sonicheroes.utils.toner` in your `Mods` folder and let the launcher re-download it (it'll happen post launcher restart).
> 
> You'll probably be good then

ok it launched but some goofy things are on the screen

![Screenshot 2023-11-20 174315](https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/88acad04-d0fd-40ce-b866-8c9419308852)

**Comment by Sewer56** [comment]
You're running the Debug Boot mod, which launches your game directly into a debug menu, this is intended. If you don't want the behaviour, disable the mod, or change its settings.

**Comment by Pepsyah** [comment]
> You're running the Debug Boot mod, which launches your game directly into a debug menu, this is intended. If you don't want the behaviour, disable the mod, or change its settings.

holy shit you are fast ok

**Comment by Pepsyah** [comment]
thx homie

**Comment by Pepsyah** [comment]
cant even get past the title screen

**Comment by Sewer56** [comment]
You probably enabled Controller Hook Custom Mapping Mod, but didn't map any keys via 'Configure Mod"

**Comment by Pepsyah** [comment]
i have the controller hook custom mapping mod turned off but i still cant get past the title screen

**Comment by Pepsyah** [comment]
maybe it has something to do with my mod order
![glublgubglbgbgi](https://github.com/Reloaded-Project/Reloaded-II/assets/104694647/56b81eec-33f9-4e73-94ea-02f201cae6e3)

**Comment by Pepsyah** [comment]
> You probably enabled Controller Hook Custom Mapping Mod, but didn't map any keys via 'Configure Mod"

wait i just turned off freecam mod and now the game is doing fine

**Comment by Sewer56** [comment]
Freecam Mod depends on Controller Hook which is configured to not accept original inputs (modded inputs only).

Hence if you're not using Custom Mapping or XInput for Controller Hook with it, controls die.
